Marlin/src/feature/pause.cpp

```diff
@@ -252,42 +260,67 @@ void wait_for_filament_reload(const int8_t max_beep_count/*=0*/) {
 }
 
 void resume_print(const float &load_length/*=0*/, const float &initial_extrude_length/*=0*/, const int8_t max_beep_count/*=0*/) {
-  bool nozzle_timed_out = false;
-
   if (!move_away_flag) return;
-
+  bool nozzle_timed_out = false;
+  bool swap_spool_disabled = true;
+  int16_t fansp=fanSpeeds[ADVANCED_PAUSE_FAN];
+  
+  #if ENABLED(ADVANCED_PAUSE_SPOOL_SWAP)
+	swap_spool_disabled= !swap_spool_enabled ; // only if next tool option enabled in lcd or gcode //steeve
+  #endif
+  
   // Re-enable the heaters if they timed out
   HOTEND_LOOP() {
     nozzle_timed_out |= thermalManager.is_heater_idle(e);
     thermalManager.reset_heater_idle_timer(e);
   }
 
-  if (nozzle_timed_out) ensure_safe_temperature();
+  // Stop blowing while heating
+  #if (ADVANCED_PAUSE_FAN < FAN_COUNT)
+	fanSpeeds[ADVANCED_PAUSE_FAN]=0; 
+  #endif
+  
+  // Heating/ensure temp of the next extruder  
+  #if ENABLED(ADVANCED_PAUSE_SPOOL_SWAP)
+  	if (swap_spool_enabled){
+	ensure_safe_temperature();
+	lcd_advanced_pause_show_message(ADVANCED_PAUSE_MESSAGE_STATUS);
+    }
+  #else
```

Comment:
Temp verification , when entering in pause , not really need , but ,it's more serious to confirm the temp is ok before to continue